src/server.js

```diff
@@ -202,6 +210,48 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`,
+  getEvent,
+  async (request, response) => {
+    const { event } = request;
+    const users = event.users || {};
+    const userInput = Object.values(users);
+
+    if (userInput.length > 0) {
+      const lat = userInput[0].lat.toString();
+      const long = userInput[0].long.toString();
+      let radius = "50000";
+      let type = "restaurant";
+      let minprice = "0";
+      let maxprice = "4";
+      const restarantData = await (
+        await fetch(
+          `https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=${lat},${long}&radius=${radius}&type=${type}&minprice=${minprice}&maxprice=${maxprice}&key=${env.API_KEY}`
```

Comment:
nit: rename `API_KEY` to `API_KEY_PLACES`, since we might be using multiple APIs in the future.

Reply:
Also, be sure to update the `.env.sample` file with `API_KEY`.

Reply:
Why is that necessary? .env.sample is in github. Would I put the actual API key?

Reply:
The .env.sample is meant to be a template for .env, so developers can look at the fields in .env.sample and know what information they need to put in the actual .env file.

I am not suggesting that you put the actual API key into the .env.sample, more like appending this line:
```
API_KEY_PLACES=
```

Reply:
alright